src/components/vm/disks/diskAdd.jsx

```diff
@@ -441,8 +441,15 @@ export class AddDiskModalBody extends React.Component {
         stateDelta.format = getDefaultVolumeFormat(pool);
         if (['dir', 'fs', 'netfs', 'gluster', 'vstorage'].indexOf(pool.type) > -1) {
             const volume = pool.volumes.find(vol => vol.name === value);
-            if (volume && volume.format)
+            if (volume && volume.format) {
                 stateDelta.format = volume.format;
+                if (volume.format === "iso") {
+                    this.onValueChanged("busType", "sata");
```

Comment:
I personally think that "scsi" should be default. Of course, both sata and scsi are valid bus types.
We already have existing code which sets "scsi" for iso files supplied thru "Custom path" option, and I think the same logic should be used for ISO storage volums.
An just reading Red Hat and IBM articles, it seems to me that scsi is more common configuration for cdrom virtual disks.

Reply:
So calling simple `this.onValueChanged("device", "cdrom");` will do the job and set default bus type to scsi. You can still of course change in the UI.

Reply:
agree